go.mod

```diff
@@ -5,38 +5,51 @@ require (
 	github.com/DataDog/zstd v1.3.5 // indirect
 	github.com/asdine/storm v0.0.0-20190216191021-fe89819f6282
 	github.com/decred/dcrd/addrmgr v1.0.2
-	github.com/decred/dcrd/blockchain/stake v1.1.0
-	github.com/decred/dcrd/chaincfg v1.4.0
-	github.com/decred/dcrd/chaincfg/chainhash v1.0.1
+	github.com/decred/dcrd/blockchain/stake/v2 v2.0.1
+	github.com/decred/dcrd/chaincfg/chainhash v1.0.2
+	github.com/decred/dcrd/chaincfg/v2 v2.2.0
 	github.com/decred/dcrd/connmgr v1.0.2
-	github.com/decred/dcrd/dcrec v0.0.0-20190402182842-879eebce3333
-	github.com/decred/dcrd/dcrutil v1.2.0
+	github.com/decred/dcrd/dcrec v1.0.0
+	github.com/decred/dcrd/dcrjson v1.2.0 // indirect
+	github.com/decred/dcrd/dcrutil v1.4.0
+	github.com/decred/dcrd/dcrutil/v2 v2.0.0
 	github.com/decred/dcrd/hdkeychain v1.1.1
 	github.com/decred/dcrd/rpcclient v1.1.0
-	github.com/decred/dcrd/txscript v1.0.3-0.20190402182842-879eebce3333
+	github.com/decred/dcrd/txscript v1.1.0
+	github.com/decred/dcrd/txscript/v2 v2.0.0
 	github.com/decred/dcrd/wire v1.2.0
 	github.com/decred/dcrdata/txhelpers v1.1.0
 	github.com/decred/dcrwallet v1.2.2
-	github.com/decred/dcrwallet/chain v1.1.1
-	github.com/decred/dcrwallet/errors v1.0.1
-	github.com/decred/dcrwallet/p2p v1.0.1
-	github.com/decred/dcrwallet/spv v1.1.1
+	github.com/decred/dcrwallet/chain v1.0.0
+	github.com/decred/dcrwallet/errors v1.1.0
+	github.com/decred/dcrwallet/p2p/v2 v2.0.0
+	github.com/decred/dcrwallet/rpc/client/dcrd v0.0.0-00010101000000-000000000000
```

Comment:
Why are we not using the current latest version `v0.0.0-20191011190505-fe6e4c8c0cce`? I see below that a replace directive is added to always use the master branch version of this module. This could cause issues down the line. Probably best to fix a version now and update later when a release is cut for the module.